server/lib/currency.ts

```diff
@@ -269,3 +270,21 @@ export function reduceArrayToCurrency(array: AmountWithCurrencyAndDate[], curren
     },
   );
 }
+
+/**
+ * Gets exchange rate between given crypto currency and fiat.
+ */
+export async function getExchangeRate(fromCurrency: string, toCurrency: string): Promise<string> {
+  try {
+    if (!SUPPORTED_CRYPTO_CURRENCIES.includes(fromCurrency) && !SUPPORTED_CRYPTO_CURRENCIES.includes(toCurrency)) {
+      return getFxRate(fromCurrency, toCurrency);
+    } else {
+      const cryptoToUSDRate = await getCryptoToUSDRate(fromCurrency);
+      const USDtoCurrencyRate = await getFxRate('USD', toCurrency);
+      return cryptoToUSDRate * USDtoCurrencyRate;
```

Comment:
I don't get the rationale, please double check.

Make sure of the behavior when `fromCurrency` is fiat and `toCurrency` crypto and add some comments to explain if that's indeed correct.

Reply:
My thinking is that we should have this function only compute the crypto exchange rate. So `fromCurrency` is always crypto and `toCurrency` is always fiat. To that end, I've changed the naming of the arguments to `cryptoCurrency` and `fiatCurrency` and also renamed the function name to `getCryptoExchangeRate`. Does this make things clearer? Also I realized we don't need the default `getFxRate` call here as again we only use the function when the `fromCurrency` is a crypto currency.

Reply:
That's not true, in the `exchangeRateQuery` we accept any currency.

Reply:
Ops sorry, you are correct. I was losing the context of this in my mind and making a mess. Anyways, I've simplified the function a bit. I think this looks better. Let me know. Thanks.